src/components/map/components/action/layer-action/modify-action/index.js

```diff
@@ -9,6 +11,9 @@ import { VlModifyAction } from "vl-mapactions/dist/vl-mapactions.js";
  *
  * @extends VlMapLayerAction
  *
+ * @property {string} [data-vl-snapping] - Attribuut wordt gebruikt om aan te geven dat er bij het bewerken snapping mag gebeuren op de meegegeven vl-map-wfs-layers.
```

Comment:
Suggestie; We proberen zoveel mogelijk Nederlandse teksten te vertalen naar het Engels in het project, zodat het toegankelijker wordt.